types/yup/index.d.ts

```diff
@@ -1,11 +1,12 @@
-// Type definitions for yup 0.26
+// Type definitions for yup 0.27
```

Comment:
Is this something that was added specifcally in yup 0.27? Does this have to be incremented?

Reply:
This is my first contribution to DefinitelyTyped, so it's possible I didn't correctly follow the protocol to the letter. One of the instructions for submitting a PR was to increment the version number. I wasn't clear on what that meant, and this was the only place a version number was evident to me.

The yup package is currently at version 0.26.6. This PR is not addressing new features, but is attempting to more correctly represent existing features. Let me know how I should proceed on this.

Reply:
I think we should leave it as 0.26, it's a bad experience for the types to mismatch the package and while there is nothing really saying that we need to match the types to the package version, it quickly reaffirms that you are using the right types version when you `npm install` it. 

Thoughts?

Reply:
DefinitelyTyped will auto increment the patch version so we can leave the major and minor version as is to keep it in step with yup